src/contracts/ParaSwapDebtSwapAdapterV3.sol

```diff
@@ -20,8 +20,23 @@ contract ParaSwapDebtSwapAdapterV3 is ParaSwapDebtSwapAdapter {
     address owner
   ) ParaSwapDebtSwapAdapter(addressesProvider, pool, augustusRegistry, owner) {}
 
-  function _getReserveData(address asset) internal view override returns (address, address) {
+  function _getReserveData(
+    address asset
+  ) internal view override returns (address, address, address) {
     DataTypes.ReserveData memory reserveData = POOL.getReserveData(asset);
-    return (reserveData.variableDebtTokenAddress, reserveData.stableDebtTokenAddress);
+    return (
+      reserveData.variableDebtTokenAddress,
+      reserveData.stableDebtTokenAddress,
+      reserveData.aTokenAddress
+    );
+  }
+
+  function _supply(
+    address asset,
+    uint256 amount,
+    address to,
+    uint16 referralCode
+  ) internal override {
+    POOL.deposit(asset, amount, to, referralCode);
```

Comment:
just curious, if handling v3 separate wouldn't it make sense to use `supply` instead (as i understand deposit is more or less deprecated)

https://github.com/aave/aave-v3-core/blob/master/contracts/interfaces/IPool.sol#L727

Reply:
Nice catch, done!

Reply:
Addressed in [8d12ee5](https://github.com/bgd-labs/aave-debt-swap/pull/17/commits/8d12ee54d096cb5dea364e787dee58d3703e1b2e)